gkgen/bcp47gen_test.go

```diff
@@ -5,72 +5,70 @@ import (
 	"strings"
 	"testing"
 
-	"github.com/stretchr/testify/suite"
+	"github.com/stretchr/testify/require"
 )
 
 type TestData struct {
 	BCP47String *string
 }
 
-// BCP47ValidatorTestSuite
-type BCP47ValidatorTestSuite struct {
-	suite.Suite
-}
-
-// TestBCP47ValidatorTestSuite
-func TestBCP47ValidatorTestSuite(t *testing.T) {
-	suite.Run(t, new(BCP47ValidatorTestSuite))
-}
-
 // TestIsBCP47_ParamsLength
-func (s *BCP47ValidatorTestSuite) TestIsBCP47_ParamsLength() {
+func TestIsBCP47_ParamsLength(t *testing.T) {
 	params := []string{"I'll break..."}
 	et := reflect.TypeOf(ExampleStruct{})
 	field, _ := et.FieldByName("BCP47String")
 	b := NewBCP47Validator()
 	_, err := b.Generate(et, field, params)
-	s.Require().Error(err)
+	require.Error(t, err)
 }
 
 // TestIsBCP47_FieldPtr
-func (s *BCP47ValidatorTestSuite) TestIsBCP47_FieldPtr() {
+func TestIsBCP47_FieldPtr(t *testing.T) {
 	et := reflect.TypeOf(ExampleStruct{})
 	field, _ := et.FieldByName("BCP47NonString")
 	b := NewBCP47Validator()
 	_, err := b.Generate(et, field, []string{})
-	s.Require().Error(err)
+	require.Error(t, err)
 }
 
 // TestIsBCP47_FieldNestedPtr
-func (s *BCP47ValidatorTestSuite) TestIsBCP47_FieldNestedPtr() {
+func TestIsBCP47_FieldNestedPtr(t *testing.T) {
 	et := reflect.TypeOf(ExampleStruct{})
 	field, _ := et.FieldByName("BCP47NonStringPtr")
 	b := NewBCP47Validator()
 	_, err := b.Generate(et, field, []string{})
-	s.Require().Error(err)
+	require.Error(t, err)
 }
 
-// TestGenerateHexValidationCode_String
-func (s *BCP47ValidatorTestSuite) TestGenerateHexValidationCode_String() {
+// TestGenerateBCP47ValidationCode_String
+func TestGenerateBCP47ValidationCode_String(t *testing.T) {
 	hv := NewBCP47Validator()
 	e := ExampleStruct{}
 	et := reflect.TypeOf(e)
 	field, _ := et.FieldByName("BCP47String")
 
 	code, err := hv.Generate(et, field, []string{})
-	s.Nil(err)
+	require.NoError(t, err)
 	code = strings.Replace(strings.TrimSpace(code), "\t", "", -1)
-	s.Equal("if err := gokay.IsBCP47(&s.BCP47String); err != nil {\nerrorsBCP47String = append(errorsBCP47String, err)\n}", code)
+	require.Equal(
+		t,
+		"if err := gokay.IsBCP47(&s.BCP47String); err != nil {\nerrorsBCP47String = append(errorsBCP47String, err)\n}",
+		code,
+	)
 }
 
-// TestGenerateHexValidationCode_StringPtr
-func (s *BCP47ValidatorTestSuite) TestGenerateHexValidationCode_StringPtr() {
+// TestGenerateBCP47ValidationCode_StringPtr
```

Comment:
Do we really need these godoc comments on every test? I know you didn't add them, but it feels like they're only here to silence IDE/linter warnings. Shall we remove them?